src/main/java/uk/gov/hmcts/reform/ccd/data/dao/CaseLinkRepository.java

```diff
@@ -12,4 +14,7 @@
 
     List<CaseLinkEntity> findByCaseId(final Long caseId);
 
+    @Query("FROM CaseLinkEntity WHERE linked_case_id IN :queryLinkedCases")
```

Comment:
No sure if the SELECT is needed in jpa?

Reply:
I believe the `SELECT` clause is inferred/implied

Reply:
👍